.github/workflows/type-quality-monitor.yml

```
@@ -0,0 +1,101 @@
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # 毎週月曜日の午前9時（JST）に実行
+    - cron: '0 0 * * 1'
+
+jobs:
+  type-quality:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)